src/Entity/Tag.php

```diff
@@ -61,4 +72,40 @@ public function __toString(): string
     {
         return $this->name;
     }
+	
+	public function getCreatedAt(): ?\DateTimeImmutable
+    {
+        return $this->createdAt;
+    }
+
+    public function setCreatedAt(\DateTimeImmutable $createdAt): self
+    {
+        $this->createdAt = $createdAt;
+
+        return $this;
+    }
+
+    public function getUpdatedAt(): ?\DateTimeImmutable
+    {
+        return $this->updatedAt;
+    }
+
+    public function setUpdatedAt(\DateTimeImmutable $updatedAt): self
+    {
+        $this->updatedAt = $updatedAt;
+
+        return $this;
+    }
+	
+	#[PrePersist]
+    #[PreUpdate]
+    public function updatedTimestamps()
+    {
+        if ($this->getUpdatedAt() == null) {
+            $this->setUpdatedAt(new \DateTimeImmutable('now'));
+        }
+        if ($this->getCreatedAt() == null) {
+            $this->setCreatedAt(new \DateTimeImmutable('now'));
+        }
```

Comment:
Sorry, I just copied from an older project. The point was to demonstrate how it can be used and should be used in an official demo. I will update shortly.